geomesa-tools/bin/install-kafka.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Apache License, Version 2.0 which
+# accompanies this distribution and is available at
+# http://www.opensource.org/licenses/apache2.0.php.
+#
+
+if [[ (-z "$1") ]]; then
+    echo "Error: Provide one arg which is the target directory (e.g. /opt/jboss/standalone/deployments/geoserver.war/WEB-INF/lib)"
+    exit
+else
+    install_dir=$1
+    NL=$'\n'
+    read -r -p "Install Kafka DataStore dependencies to ${install_dir}?${NL}Confirm? [Y/n]" confirm
+    confirm=${confirm,,} #lowercasing
+    if [[ $confirm =~ ^(yes|y) ]]; then
+        # get stuff
```

Comment:
(I copied the install-hadoop-accumulo.sh file;))

I think the rst is clear.... look below.